src/Data/Rational/Properties.agda

```diff
@@ -880,6 +971,71 @@ private
   { isCommutativeRing = +-*-isCommutativeRing
   }
 
+------------------------------------------------------------------------
+-- Properties of _*_ and -_
+
+neg-distribˡ-* : ∀ p q → - (p * q) ≡ - p * q
```

Comment:
It would be nice if these two proofs could be lifted to the [RingMonomorphism module](https://github.com/agda/agda-stdlib/blob/master/src/Algebra/Morphism/RingMonomorphism.agda) and then used from there by constructing the `+-*-ringMonomorphism`?